app/serializable/serializable_customer_resource.rb

```diff
@@ -1,11 +1,80 @@
 class SerializableCustomerResource < SerializableResource
   type 'customerResources'
 
-  attributes :title_id, 
-             :vendor_id, 
-             :package_id, 
-             :package_name, 
-             :is_selected, 
-             :selected_count, 
-             :title_count
+  attributes :contributors,
+             :coverage_statement,
+             :custom_coverages,
+             :custom_embargo_period,
+             :description,
+             :is_peer_reviewed,
+             :is_selected,
+             :managed_coverages,
+             :managed_embargo_period,
+             :name,
+             :package_id,
+             :package_name,
+             :publisher_name,
+             :subjects,
+             :title_id,
+             :url,
+             :vendor_id,
+             :vendor_name,
+             :visibility_data
+
+  attribute :identifiers do
```

Comment:
This makes sense doing it in the serializer. We should also do the same for titles and package's `contentType`